src/app/components/auth/AuthInitSignUp.tsx

```diff
@@ -95,8 +115,22 @@ const AuthInitSignUp: React.FC = () => {
       <Form
         methods={methods}
         onSubmit={methods.handleSubmit(onSubmit)}
-        className={classNames(isConfirming && 'hide')}
+        className={classNames('row', { hide: isConfirming })}
```

Comment:
`<Form />` probably is already rendering the `row` class by default. Check if it does, and if so, remove the `row` from here.